examples/Project.toml

```diff
@@ -9,11 +9,11 @@ RegularizedOptimization = "196f2941-2d58-45ba-9f13-43a2532b2fa8"
 RegularizedProblems = "ea076b23-609f-44d2-bb12-a4ae45328278"
 
 [compat]
-ADNLPModels = "0.4"
+ADNLPModels = "0.6"
```

Comment:
```suggestion
ADNLPModels = "0.7"
```

Reply:
I think changing the ADNLPModels compat here should work, and adding NLPModels 0.20 to the Project.toml of the repo (but I don't think I can make a code suggestion in this PR since it is not a modified file).